test/test_ppx.ml

```diff
@@ -137,6 +137,15 @@ let basics = "ppx basics", HtmlTests.make Html.[
   [[%html "<ol>   <li>foo</li>  <li>bar</li>   </ol>"]],
   [ol [li [txt "foo"] ; li [txt "bar"]]] ;
 
+  "whitespace in tr",
+  [[%html "<tr>   <td>foo</td>    <td>bar</td>   </tr>"]],
+  [tr [td [txt "foo"] ; td [txt "bar"]]] ;
+
+  "whitespace in table",
+  [[%html "<table>    <tr><td>foo</td></tr>   <tr><td>bar</td></tr>   </table>"]],
+  (* [table [tr [td [txt "foo"]] ; tr [td [txt "bar"]]]] ; *)
```

Comment:
`table [tr [td [txt "foo"]] ; tr [td [txt "bar"]]]` should work (without a `tbody`).

Reply:
Doing this I get `expecting
<table><tbody><tr><td>foo</td></tr><tr><td>bar</td></tr></tbody></table>, got
<table><tr><td>foo</td></tr><tr><td>bar</td></tr></table>.`. What did I miss ?

Reply:
Hm, the combinators and the syntax should produce exactly the same thing. This sounds like a bug.